faq.rst

```diff
@@ -399,62 +290,38 @@ How do I compare my local repository to a remote repository?
 To display the list of changes that are in your local repository, but not
 in the remote, use::
 
- hg outgoing <remote repository>
+    $ git log origin/master..
 
 This is the list of changes that will be sent if you call
-``hg push <remote repository>``.  It does **not** include any :ref:`uncommitted
+``git push``.  It does **not** include any :ref:`uncommitted
 changes <hg-status>` in your working copy!
 
 Conversely, for the list of changes that are in the remote repository but
 not in the local, use::
```

Comment:
[Sorry, my previous comment was at the wrong place]

This only lists commits that were copied from the remote the last time the local copy was updated (via “git pull” or “git fetch”). I don’t think it will list new commits that will be fetched.